Source/Scene/Scene.js

```diff
@@ -2787,7 +2840,7 @@ define([
         }
     }
 
-    function callAfterRenderFunctions(frameState) {
+    function callAfterRenderCycleFunctions(frameState) {
```

Comment:
Why the name change?

Reply:
I wanted to make it clear that it was called every call to `scene.render`, whether a new frame is rendered or not, but it's not necessary in retrospect.